File: tests/transpiler/x/pas/save_jsonl_stdout.pas
{$mode objfpc}
program Main;
uses SysUtils;
type Anon1 = record
  name: string;
  age: integer;
end;
var
  people: array of Anon1;
begin
  people := [(name: 'Alice'; age: 30), (name: 'Bob'; age: 25)];
  for row in people do begin
  writeln(((((((('{' + '"name": ') + '"') + row.name) + '"') + ', ') + '"age": ') + IntToStr(row.age)) + '}');
end;
end.

File: archived/tests/transpiler/x/php/save_jsonl_stdout.php
<?php
$people = [["name" => "Alice", "age" => 30], ["name" => "Bob", "age" => 25]];
foreach ($people as $_row) {
  $j = json_encode($_row);
  $j = str_replace(":", ": ", $j);
  $j = str_replace(",", ", ", $j);
  echo $j . PHP_EOL;
}
;
